<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>EigenGridsDlgUI</class>
  <widget class="QDialog" name="EigenGridsDlgUI" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>272</width>
        <height>291</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>EigenGrids</string>
    </property>
    <widget class="Q3GroupBox" name="groupBox1" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>10</y>
          <width>250</width>
          <height>230</height>
        </rect>
      </property>
      <property name="title" >
        <string>Controls</string>
      </property>
      <widget class="QLabel" name="textLabel1" >
        <property name="geometry" >
          <rect>
            <x>14</x>
            <y>195</y>
            <width>40</width>
            <height>20</height>
          </rect>
        </property>
        <property name="text" >
          <string>Type:</string>
        </property>
        <property name="wordWrap" >
          <bool>false</bool>
        </property>
      </widget>
      <widget class="QComboBox" name="typeBox" >
        <property name="geometry" >
          <rect>
            <x>60</x>
            <y>194</y>
            <width>111</width>
            <height>21</height>
          </rect>
        </property>
      </widget>
      <widget class="QSlider" name="slider1" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>20</y>
            <width>221</width>
            <height>24</height>
          </rect>
        </property>
        <property name="orientation" >
          <enum>Qt::Horizontal</enum>
        </property>
      </widget>
      <widget class="QSlider" name="slider2" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>40</y>
            <width>221</width>
            <height>24</height>
          </rect>
        </property>
        <property name="orientation" >
          <enum>Qt::Horizontal</enum>
        </property>
      </widget>
      <widget class="QSlider" name="slider3" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>60</y>
            <width>221</width>
            <height>24</height>
          </rect>
        </property>
        <property name="orientation" >
          <enum>Qt::Horizontal</enum>
        </property>
      </widget>
      <widget class="QSlider" name="slider4" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>80</y>
            <width>221</width>
            <height>24</height>
          </rect>
        </property>
        <property name="orientation" >
          <enum>Qt::Horizontal</enum>
        </property>
      </widget>
      <widget class="QSlider" name="slider5" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>100</y>
            <width>221</width>
            <height>24</height>
          </rect>
        </property>
        <property name="orientation" >
          <enum>Qt::Horizontal</enum>
        </property>
      </widget>
      <widget class="QSlider" name="slider6" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>120</y>
            <width>221</width>
            <height>24</height>
          </rect>
        </property>
        <property name="orientation" >
          <enum>Qt::Horizontal</enum>
        </property>
      </widget>
      <widget class="QPushButton" name="loadButton" >
        <property name="geometry" >
          <rect>
            <x>10</x>
            <y>150</y>
            <width>50</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text" >
          <string>Load</string>
        </property>
      </widget>
      <widget class="QPushButton" name="loadMeanButton" >
        <property name="geometry" >
          <rect>
            <x>80</x>
            <y>150</y>
            <width>80</width>
            <height>30</height>
          </rect>
        </property>
        <property name="text" >
          <string>Load Mean</string>
        </property>
      </widget>
      <widget class="QPushButton" name="clearButton" >
        <property name="geometry" >
          <rect>
            <x>181</x>
            <y>150</y>
            <width>50</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text" >
          <string>Clear</string>
        </property>
      </widget>
      <widget class="QPushButton" name="goButton" >
        <property name="geometry" >
          <rect>
            <x>190</x>
            <y>190</y>
            <width>41</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text" >
          <string>Go!</string>
        </property>
      </widget>
    </widget>
    <widget class="QPushButton" name="exitButton" >
      <property name="geometry" >
        <rect>
          <x>190</x>
          <y>250</y>
          <width>71</width>
          <height>31</height>
        </rect>
      </property>
      <property name="text" >
        <string>Exit</string>
      </property>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="global" >vector</include>
    <include location="local" >dataTypes.h</include>
  </includes>
  <connections>
    <connection>
      <sender>exitButton</sender>
      <signal>clicked()</signal>
      <receiver>EigenGridsDlgUI</receiver>
      <slot>exitButton_clicked()</slot>
    </connection>
    <connection>
      <sender>loadButton</sender>
      <signal>clicked()</signal>
      <receiver>EigenGridsDlgUI</receiver>
      <slot>loadButton_clicked()</slot>
    </connection>
    <connection>
      <sender>loadMeanButton</sender>
      <signal>clicked()</signal>
      <receiver>EigenGridsDlgUI</receiver>
      <slot>loadMeanButton_clicked()</slot>
    </connection>
    <connection>
      <sender>clearButton</sender>
      <signal>clicked()</signal>
      <receiver>EigenGridsDlgUI</receiver>
      <slot>clearButton_clicked()</slot>
    </connection>
    <connection>
      <sender>goButton</sender>
      <signal>clicked()</signal>
      <receiver>EigenGridsDlgUI</receiver>
      <slot>goButton_clicked()</slot>
    </connection>
  </connections>
</ui>
